src/main/frontend/src/stores/utils.js

```diff
@@ -9,17 +9,7 @@ function parseIfJson(val){
 }
 
 export function createLocalStore(key, initialValue) {
-  let localValue = initialValue;
-
-  if (process.browser && localStorage.getItem(key)) {
-    let maybeStoredValue = parseIfJson(localStorage.getItem(key));
-    if (!isNaN(maybeStoredValue)) {
-      if ((typeof maybeStoredValue === 'number' && isFinite(maybeStoredValue)) ||
-          (Object.prototype.toString.apply(maybeStoredValue) === '[object Number]')) {
-          localValue = Number(maybeStoredValue);
-      }
-    }
-  }
+	const localValue = process.browser && localStorage.getItem(key) ? parseIfJson(localStorage.getItem(key)) : initialValue;
```

Comment:
It seems that you removed code that I added last week to prevent `NaN` on some unknown `localStorage` state. The `NaN` was being appended to the end of the generation path rather than the next index.

Reply:
yes, I remembered that so I transformed it into Number when getting from localStorage @VaughnVernon

Reply:
Are you referring to this code?

```
const tempProjectGenerationIndex = Number($projectGenerationIndex)
```

If I understand correctly, `NaN` *is a* `Number` and thus will not be converted to a different Number value. That's what was (partially) wrong with the implementation previous to my change.